src/bootstrap.scripts.loader.js

```diff
@@ -0,0 +1,20 @@
+/* eslint func-names: 0 */
+import loaderUtils from 'loader-utils';
+
+import config from './bootstrap.config';
+import createBootstrapRequire from './utils/createBootstrapRequire';
+
+module.exports = function() {
+  if (this.cacheable) this.cacheable();
+
+  const { bootstrapPath } = loaderUtils.parseQuery(this.query);
+
+  const scriptsModules = Object.keys(config.scripts);
+
+  return (
+    scriptsModules
+      .filter(module => config.scripts[module])
```

Comment:
We filter only what user said to filter, it's his choice. He can easily define additional file in his config and it will be added to bundle.